lib/spack/spack/build_environment.py

```diff
@@ -553,6 +556,22 @@ def get_std_cmake_args(pkg):
     return spack.build_systems.cmake.CMakePackage._std_args(pkg)
 
 
+def get_std_meson_args(pkg):
```

Comment:
I think we _might_ want to avoid this. The CMake equivalent was added back in the day before separate build systems existed, and was only retained because there are still many CMake packages that haven't been converted to the `CMakePackage` build system. The only reason we would want this is if we had a package that used Meson for some versions and another build system for others.

Reply:
This was added late due to having to use a Package for an application (atk) that is converting from AutotoolsPackage to MesonPackage.  If we want to maintain history, then we have to do both.